pipelines/tekton/aiedge-e2e/aiedge-e2e.pipelinerun.yaml

```diff
@@ -4,54 +4,56 @@ metadata:
   labels:
     tekton.dev/pipeline: aiedge-e2e
   generateName: aiedge-e2e-
+  annotations:
```

Comment:
You don't need to define the duplicate annotations in the PipelineRuns.

Labels and annotations will [propagate](https://tekton.dev/docs/pipelines/labels/) from `Pipeline -> PipelineRun -> TaskRun`  with any labels on annotations defined in the `*Run` object taking precedence.

Reply:
Fixed, thanks.